modules/auxiliary/scanner/http/titan_ftp_admin_pwd.rb

```diff
@@ -89,15 +89,39 @@ def run_host(ip)
         print_good("#{ip}:#{datastore['RPORT']} - Base Directory: #{info[:basedir]}")
       end
       print_good("#{ip}:#{datastore['RPORT']} - Admin Credentials: '#{info[:username]}:#{info[:password]}'")
-      report_auth_info(
-        :host       => ip,
-        :port       => datastore['RPORT'],
-        :user       => info[:username],
-        :pass       => info[:password],
-        :ptype      => "password",
-        :proto      => "http",
-        :sname      => "Titan FTP Admin Console"
+      report_cred(
+        ip: ip,
+        port: 21,
```

Comment:
Took a look, so the module is targeting the Admin interface, not FTP. It does have the admin port, but it should be the same as ```datastore['RPORT']```, so I'll just use that.

Merging!